tests/e2e/inversify.config.ts

```diff
@@ -132,5 +134,5 @@ e2eContainer.bind<GitOauthAppsSettings>(CLASSES.GitOauthAppsSettings).to(GitOaut
 e2eContainer.bind<AnimationChecker>(CLASSES.AnimationChecker).to(AnimationChecker);
 e2eContainer.bind<Sanitizer>(CLASSES.Sanitizer).to(Sanitizer);
 e2eContainer.bind<NavigationBar>(CLASSES.NavigationBar).to(NavigationBar);
-
+e2eContainer.bind<IApiUrlResolver>(CLASSES.ApiUrlResolver).to(ApiUrlResolver);
```

Comment:
So sorry, I missed this, shouldn't the IApiUrlResolver be bound to a TYPE?
e.g.:
```
e2eContainer.bind<ITestWorkspaceUtil>(TYPES.WorkspaceUtil).to(TestWorkspaceUtil);
```